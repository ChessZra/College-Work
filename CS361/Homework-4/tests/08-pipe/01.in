/bin/echo This is a test|sed s/test/CS361/g
