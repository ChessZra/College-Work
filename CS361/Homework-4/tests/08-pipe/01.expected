cs361sh> This is a CS361
cs361sh> 
